### PR TITLE
AG-11297 Navigator DOM fallback slider

### PR DESCRIPTION
**Pull request by olegat**
https://ag-grid.atlassian.net/browse/AG-11297

### CODE REVIEW

File: packages/ag-charts-community/src/util/number.ts [hunk 1]
```diff
@@ -60,3 +60,20 @@ export function countFractionDigits(value: number, maximumFractionDigits = 10) {
         .split('.');
     return decimal.length;
 }
+
+/**
+ * @param ratio A number from 0 to 1.
+ * @param locale The locale to use to format the number.
+ */
+export function formatNormalizedPercentage(ratio: number, locale?: string) {
+    locale = locale || navigator.language;
```
Comment (alantreadway) [inline packages/ag-charts-community/src/util/number.ts]: Nit: Prefer `??` here?

**Review by alantreadway — COMMENTED** [review]
LGTM, just one minor nit

File: packages/ag-charts-community/src/util/number.ts (hunk 1 above)
Comment (olegat) [reply packages/ag-charts-community/src/util/number.ts]: done.

**Review by alantreadway — APPROVED** [review]
LGTM

**Pull request merged**